### ISSUE TITLE
SCC becomes negative

### ISSUE THREAD

**Issue by Myblackcat0216** [CR-Gjx/SCC#3]
Hello, I also have a similar problem, my SCC is  negative during training.I set the batch_size to be 1 and lambda_scc to be 5 following the setting suggested in the paper.

**Comment by CR-Gjx** [comment]
Hello, can you provide more experimental settings? e.g. datasets/ sample generated images/ experimental logs. Importantly, please set the batch_size to be 1, and try to vary lambda_scc to be smaller?

**Comment by Myblackcat0216** [comment]
Sorry, I just saw this email, my datasets trainA and trainB have 506 images each.I am using the SCC loss function on a derivative project of cyclegan. The main task is the mutual conversion between the two domains during the day and night. The training log has not been saved yet, and it will be sent to you after training.

**Comment by CR-Gjx** [comment]
I see, I tried my loss on Day and Night datasets, and the performance is good with lambda_SCC=5. Can you try using my cyclegan script directly and set batch_size =1 ?

**Comment by Myblackcat0216** [comment]
Ok, I'll try again. Thank you very much for your help

**Comment by junzhin** [comment]
Hi, I have a followed-up question, if we set the training batch size to be one during the training, do we need to adjust the learning rate accordingly?

**Comment by CR-Gjx** [comment]
I never tried to adjust the learning rate. lol

**Comment by CR-Gjx** [comment]
Problem solved, I will close this issue.
